include/detchannelmaps/TPCChannelMap.hpp

```diff
@@ -67,11 +67,10 @@ class TPCChannelMap
   virtual uint get_offline_channel_from_crate_slot_stream_chan(uint crate, uint slot, uint stream, uint channel) {
     
     constexpr uint n_chan_per_stream = 64;
-    constexpr uint n_streams_per_link = 4;
 
-    uint link = (stream >> 6);
-    uint substream = (stream & 0x3f);
-    uint ch = n_chan_per_stream*substream+channel;
+    uint link = (stream >> 6) & 1;
```

Comment:
Without a strict argument che k, these reduced masks are likely to hide issues with out-of-range arguments. Phr could either use the full bit ask for the link/substream fields or add a strict argument check

Reply:
Added explicit check on stream number. I think the better way would be to actually use this stream number in the text file maps, but that will be another iteration with offline.